app/models/concerns/orchestration.rb

```diff
@@ -151,6 +151,18 @@ def process(queue_name)
     fail_queue(q)
 
     rollback
+  ensure
+    unless q.nil?
+      logger.info("Processed queue '#{queue_name}', #{q.completed.count}/#{q.all.count} completed tasks") unless q.empty?
+      q.all.each do |task|
+        msg = "Task #{task.name} *#{task.status}*"
+        if task.status !~ /^(completed|pending)$/
```

Comment:
No need to add a regexp in here, use a simple boolean please, or `if ['completed', 'pending'].include?(task.status)` (and flip the order of the conditional is the matching block, then the false block instead of negating it).